iDiff/differs/aptDiff.go

```diff
@@ -0,0 +1,94 @@
+package differs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// AptDiff compares the packages installed by apt-get.
+func AptDiff(img1, img2 string) (string, error) {
+	pack1, err := getPackages(img1)
+	if err != nil {
+		return "", err
+	}
+	pack2, err := getPackages(img2)
+	if err != nil {
+		return "", err
+	}
+
+	diff1, diff2 := diffMaps(pack1, pack2)
+	s1 := fmt.Sprintf("Image %s had the following packages which differed:\n%s\n", img1, strings.Join(diff1, "\n"))
+	s2 := fmt.Sprintf("\nImage %s had the following packages which differed:\n%s", img2, strings.Join(diff2, "\n"))
+	return s1 + s2, nil
+}
+
+func diffMaps(map1, map2 map[string]string) ([]string, []string) {
```

Comment:
Try to implement this in such a way that more than one value is storable with a package.  You can consider how a user might want to know the filesize of the packages.  Perhaps you can make a struct, `PackageInfo` and the map is key->struct